mailpoet/lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php

```diff
@@ -211,4 +195,36 @@ private function getEmailVolumeLimitReachedMessage(): string {
 
     return "{$message}<br/>";
   }
+
+  /**
+ * Returns error $message and $operation for API::RESPONSE_CODE_CAN_NOT_SEND
+ */
```

Comment:
I have noticed that this comment is formatted incorrectly. Please, can you fix it?